docs/technical-reference/api/moduleTemplate-cr.md

```diff
@@ -87,6 +87,11 @@ The core of any ModuleTemplate CR, the descriptor can be one of the schemas ment
 
 By default, it will most likely be easiest to use [Kyma CLI](https://github.com/kyma-project/cli/tree/main) and its `create module` command to create a template with a valid descriptor, but it can also be generated manually, for example using [OCM CLI](https://github.com/open-component-model/ocm/tree/main/cmds/ocm).
 
+### `.spec.mandatory`
+
+The `mandatory` field indicates whether the module is installed in all runtime clusters without any interaction from the user.
```

Comment:
```suggestion
The `mandatory` field indicates whether the module is installed in all runtime clusters without any interaction from the user.
```

That is wrong, it is not about that it `is` installed, but it `should be` installed. I think `is` is very confusing here. That mandatory field does not indicate the current state, but how the module template should be treated as